README.md

```diff
@@ -209,7 +209,7 @@ that you have a 32bit version of Python installed.
 Populate your database:
 
     $ ./manage.py sync_couch_views
-    $ ./manage.py migrate --noinput
+    $ env CCHQ_IS_FRESH_INSTALL=1 ./manage.py migrate --noinput
```

Comment:
just a couple things:
- we should mention explicitly that you don't want to run migrate this way every time, just the first time
- this will work for linux, but not for windows (not sure about mac). we might want to include the commands for the different os's as well, or just mention that you need a temporary environment variable named `CCHQ_IS_FRESH_INSTALL` to be set to '1' in order for this to work